02-express-tutorial/routes/people.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+
+const { getPeople, addPerson, getPersonById, updatePerson, deletePerson } = require("../controllers/people");
+
+router.get("/", getPeople); 
+router.post("/", addPerson); 
+router.get("/:id", getPersonById); 
+router.put("/:id", updatePerson); 
```

Comment:
While `PUT` vs. `PATCH` are sometimes used interchangeably, strictly speaking the behavior of `updatePerson` is more inline with `PATCH`.

Reference: https://www.geeksforgeeks.org/difference-between-put-and-patch-request/